product_variant_supplierinfo/models/product_product.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# © 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2016 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import api, models, fields
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    seller_ids = fields.Many2many(
+        'product.supplierinfo',
+        compute='_compute_seller_ids')
+    seller_delay = fields.Integer(
```

Comment:
I have checked again and I have to redefine this field.
Indeed the field "seller_delay" and "seller_id" must return the supplier of the "variant" and ot the supplier of the template.
I you show the field seller_id on the product product and remove my code, the wrong seller_id is returned. The help/string is needed as the value is not the same as the related